### DISCUSSION TITLE
Blocks tab

### DISCUSSION THREAD

**Discussion by Admiralgogo** [ScratchAddons/ScratchAddons#6703] [Ideas]
Like a tab where you can use blocks submitted by other people, such as like better talk where it types itself out.
idk what to say

**Comment by CubesterYT** [comment]
Scratch doesn't support custom extensions. It is only supported by Scratch ***mods***, such as TurboWarp, which has a dedicated section for custom user submitted extensions.

**Reply by CST1229** [reply]
I think they mean like a way to find and add custom blocks (as in, the My Blocks blocks), which projects can already be used for.

**Comment by DNin01** [comment]
And how would one submit a block?

**Comment by BroJac5246** [comment]
So basically a Libraries feature? I would love to see this! But how would blocks be added? Would library creators have to apply, or would you add a project ID and all relevant blocks are automatically added?

Related: #4801 (and I could swear there was somewhere else but I couldn't find it)

**Reply by WorldLanguages** [reply]
Yeah, I think there's a 3 digit issue related to this somewhere...
